src/waveform/renderers/waveformmark.cpp

```diff
@@ -67,6 +67,17 @@ float overlappingMarkerIncrement(const float labelRectHeight, const float breadt
     return std::max(minIncrement, fullIncrement - std::max(0.f, threshold - breadth));
 }
 
+bool isShowUntilNextPositionControl(const QString& positionControl) {
+    // To identify which markers are included in the beat/time until next marker
+    // display, in addition to the hotcues
+    static const QStringList list{{"cue_point",
+            "intro_start_position",
+            "intro_end_position",
+            "outro_start_position",
+            "outro_end_position"}};
+    return list.contains(positionControl);
```

Comment:
Something like that (untested) 
```suggestion
    constexpr std::array<QLatin1String, 5>list{{"cue_point",
            "intro_start_position",
            "intro_end_position",
            "outro_start_position",
            "outro_end_position"}};
    auto it = std::find(list.cbegin(), list.cend() ,positionControl);
    return it != arr.cend();
```

Reply:
using namespace Qt::Literals::StringLiterals;
constexpr std::array list = {"cue_point"_L1,
            "intro_start_position"_L1,
            "intro_end_position"_L1,
            "outro_start_position"_L1,
            "outro_end_position"_L1};
    return std::any_of(list.cbegin(), list.cend(), [](auto& view){
        view == positionControl;
}};

Reply:
nice!